Configuration/Services.yaml

```diff
@@ -24,7 +28,17 @@ services:
   JWeiland\Reserve\DataHandler\FacilityClearCacheAfterUpdate:
     public: true
 
-  # Called by makeInstance in our PeriodRegistrationViewHelper
+  # Make all of our Services public
```

Comment:
If you want to make all services public use that one here: https://github.com/jweiland-net/events2/blob/main/Configuration/Services.yaml#L74-L77